pydra/engine/auxiliary.py

```diff
@@ -1,7 +1,10 @@
-import pdb
```

Comment:
I like keeping this on the files, at least when we are changing the code so much.

Reply:
if you're using python 3.7, we can use the new builtin `breakpoint()` instead of `import pdb ... pdb.set_trace()`. then we don't have to import `pdb` and have that be committed. what do you think?

Reply:
I'm using various python. you can remove it if you want, but I won't promise that I won't add it (and forget to remove) again, I'm using pdb all the time. 

And I'm happy to remove before release

Reply:
I'll put in a vote for removing `pdb` and `set_trace` (and `breakpoint`, in Py37) from the main tree.

Reply:
just a note, we don't want to have py37 only. So I'm testing on py36 or py35 and I think that's fine